src/base/bittorrent/torrentinfo.cpp

```diff
@@ -205,7 +205,7 @@ qlonglong TorrentInfo::totalSize() const
     return m_nativeInfo->total_size();
 }
 
-int TorrentInfo::filesCount() const
+int TorrentInfo::filesCount() const noexcept
```

Comment:
I wouldn't use `noexcept` unless you checked that all function calls used here are guaranteed not to throw (best marked with `noexcept`), otherwise the `noexcept` contract can be easily broken and become pointless.

Reply:
Ok.
Comments are addressed.